docs/lua-api/global-functions/registerhook.md

```diff
@@ -27,7 +29,18 @@ Returns two ids, both of which must be passed to `UnregisterHook` if you want to
 
 ## Example
 ```lua
-RegisterHook("/Script/Engine.PlayerController:ClientRestart", function()
-    print("PlayerController restarted\n")
+PreId, PostId = RegisterHook("/Script/Engine.PlayerController:ClientRestart", function(Context, NewPawn)
+    local PlayerController = Context:get()
+    local Pawn = NewPawn:get()
+
+    print("PayerController FullName: " .. PlayerController:GetFullName())
+    if Pawn:IsValid() then
+        print("NewPawn FullName: " .. Pawn:GetFullName())
+    end
```

Comment:
```suggestion
    print("PayerController FullName: " .. PlayerController:GetFullName() .. "\n")
    if Pawn:IsValid() then
        print("NewPawn FullName: " .. Pawn:GetFullName() .. "\n")
    end
```

Our `print` function requires an explicit new line.
Our other examples mostly use string.format btw, since you mentioned you wanted to keep to what we already do.